lib/agent.js

```diff
@@ -126,6 +126,9 @@ Agent.prototype._getStats = function () {
   const stats = {
     frameCache: frameCacheStats
   }
+  if (this._instrumentation._runCtxMgr && this._instrumentation._runCtxMgr._runContextFromAsyncId) {
+    stats.runContextFromAsyncIdSize = this._instrumentation._runCtxMgr._runContextFromAsyncId.size
+  }
```

Comment:
REVIEW NOTE: This internal state might be helpful in manually debugging with users that are seeing high memory usage. If this value is and stays very high, then it can indicate async tasks that outlive current transactions and possibly indicate async task leaks in the application code.